### DISCUSSION TITLE
EoS System Tray

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#615] [Q&A] [answered]
I'm guessing this is an artifact of the Wayland translation, but I'm getting no System Tray icons in v8.0.

The two apps that should have them are 

1. Variety Wallpaper Changer
2. Pulse SMS

Neither do. The issue has been posted on the Variety Wallpaper Github.

Thoughts?

**Comment by danirabbit** [comment, marked as answer]
please see this discussion post: https://github.com/elementary/wingpanel/discussions/360

TL;DR There isn’t a freedesktop specification for application indicators so elementary doesn’t support this

closing as duplicate
